### DISCUSSION TITLE
whether to use gh-pages or master branch

### DISCUSSION THREAD

**Discussion by therajat08** [community/community#22599] [Other Feature Feedback, Questions, & Ideas]
<p>So i was trying to use github pages feature. In a youtube tutorial the guy said that we need to commit only on the gh-pages branch. My question is : whether to use gh-pages or master branch? </p>

**Comment by Wabri** [comment, marked as answer]
<p>usually the master is used to develop the code, while gh-pages only serves for the github pages feature. It is not a rule it is only to have a distinction between development and documentation. You can put everything in the same branch, but more you’ll develop in the repository the more confusion you’ll have with just one branch.</p>
<p>-Gabriele-</p>
